### ISSUE TITLE
Cannot sign in with google

### ISSUE THREAD

**Issue by PhilippK233** [Rov3r/scmusicplus#4]
Possible that SoundCloud blocked the ipa?

**Comment by danpasecinic** [comment]
Nope, it still works. You have got a couple of choices (personal preference) to make it to work on iOS 16 and higher versions:

- Pair your Google Account with your existing SoundCloud account
- Use your Google Account with a new SoundCloud account
- Pair your Facebook Account with your existing SoundCloud account
- Use your Facebook Account with a new SoundCloud account

From here, sign into the app store app and continue the sideloading steps like normal.

**Comment by Rov3r** [comment]
Using Sideloadly's Sideload Spoofer will fix any and all sign-in issues.
